app/views/surveys/show.html.erb

```diff
@@ -23,9 +23,9 @@
 
   <%= link_to "Take the survey", new_survey_response_path(@survey), class: 'button' %>
   <%= link_to 'Edit', edit_survey_path(@survey), class: 'button' %>
-  <%= link_to 'Back', surveys_path, class: 'button' %>
-  <%= render partial: "surveys/export_form", locals: { survey: @survey } %>
-
+  <p>
```

Comment:
This button could do with a little bit of margin-top (and possibly remove margin-left). Just check that you don't change the styling on the other use of the popup on the index page.

Reply:
Thanks!